docs/source/deployorderer/ordererplan.md

```diff
@@ -0,0 +1,90 @@
+# Planning for an ordering service
+
+Audience: Architects, network operators, users setting up a production Fabric network who are familiar with Transport Layer Security (TLS), Public Key Infrastructure (PKI) and Membership Service Providers (MSPs).
+
+Check out the conceptual topic on [The Ordering Service](./orderer/ordering_service.html) for an overview on ordering service concepts, implementations, and the role an ordering service plays in a transaction.
+
+In Hyperledger Fabric network, a node or collection of nodes together form what's called an "ordering service", which literally orders transactions into blocks and sends them to peers to be committed to their ledgers. This separates Fabric from other distributed blockchains, such as Ethereum and Bitcoin, in which this ordering is done by any and all nodes, requiring the establishment of probabilistic consensus algorithms which eventually guarantee ledger consistency to a high degree of probability, but which are still vulnerable to divergent ledgers (also known as a ledger “fork”), where different participants in the network have a different view of the accepted order of transactions.
```

Comment:
```suggestion
In a Hyperledger Fabric network, a node or collection of nodes together form what's called an "ordering service", which literally orders transactions into blocks and sends them to peers to be committed to their ledgers. This separates Fabric from other distributed blockchains, such as Ethereum and Bitcoin, in which this ordering is done by any and all nodes, requiring the establishment of probabilistic consensus algorithms which eventually guarantee ledger consistency to a high degree of probability, but which are still vulnerable to divergent ledgers (also known as a ledger “fork”), where different participants in the network have a different view of the accepted order of transactions.
```

Reply:
`which literally orders transactions into blocks`

Side silly note: wouldn't be an accurate reflection of the 2020 lexicon without the usage (and thankfully accurate usage) of `literally` in this doc. ;)

Reply:
Haha. It felt weird to not throw that in there because you'd just be saying than an orderer orders something.